src/js/modules/sections/reducer.js

```diff
@@ -1,135 +1,131 @@
 const initialState = {
-  isFetching: false,
-  isFetched: false,
-  error: null,
-  request: null,
   sections: {
-    "news": {
+    0: {

```

Comment:
Can you explain why you changed the keys to numbers instead of sectionSlugs? Wouldn't it be harder to index into the sections State?